app/templates/request-code-select-method.html

```diff
@@ -60,35 +60,6 @@
         'title': question_title,
     }) -%}
 
-        {%- if case_type == 'HH' and request_type != 'individual-code' -%}
-
-            {%- set content -%}
-                {%- autoescape false -%}
-                    <p>{{ _('If you need to answer separately from the people you live with, you can %(open)srequest an individual access code%(close)s to start a separate census.', open='<a href="%s">' % request_individual_code_link, close='</a>')|safe }}</p>
-                {%- endautoescape -%}
-                {%- call onsPanel({
-                        'type': 'warn'
-                    })
-                -%}
-                <p><strong>{{ _('Someone in your household must still complete a census using a household access code')}}</strong></p>
-                {%- endcall -%}
-            {%- endset -%}
-
-            {%- call onsCollapsible({
```

Comment:
Can remove import of onsCollapsible component above.

Reply:
Removed call